test-data/unit/check-modules.test

```diff
@@ -2627,3 +2627,35 @@ import foo.bar.baz
 reveal_type(foo.bar.baz.x)  # E: Revealed type is 'builtins.int'
 [file foo/bar/baz.py]
 x = 0
+
+[case testModuleGetAttrAssignUnannotated]
```

Comment:
What about testing a package with a missing explicit submodule (`__getattr__` in `__init__`) and then assigning something to a submodule attribute? Or is this still unsupported?

Reply:
OK, I added two tests for this.